Assets/Scripts/Game/Leveling/AlignmentSystem.cs

```diff
@@ -0,0 +1,78 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
```

Comment:
This seems to be unused, can probably remove. Top two are also unused but Unity auto-adds those in new scripts so up to you whether to keep.